### DISCUSSION TITLE
Pull Sharks disappeared from profile?

### DISCUSSION THREAD

**Discussion by Plaenkler** [community/community#64590] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Until yesterday, I had the 2x/3x Pull Shark and today it is simply no longer visible on my profile. People I collaborate with professionally and privately still have their sharks. 😞 What could be the reason for this?

**Comment by arafajs** [comment]
Try to enable this.
![178159180-8561df6f-08a8-4053-bd32-74f913311f6d](https://github.com/community/community/assets/24510344/3345a627-4494-43eb-a906-5e8c40e73bbe)

**Reply by Plaenkler** [reply]
Do you mean the Contribution settings? It's already set to private and public contributions.

![grafik](https://github.com/community/community/assets/60503970/d27ce05e-f92a-4148-ba1b-bf02c5ea0d7a)

![grafik](https://github.com/community/community/assets/60503970/48b63255-530a-4e76-80f1-d51bc2037c30)

**Comment by IvanPotgieter01** [comment, marked as answer]
Go to [https://github.com/{username}?tab=achievements](https://github.com/%7Busername%7D?tab=achievements)

replace {username} with your username

There's an eye icon at the top left, turn it on

**Reply by Plaenkler** [reply]
In fact, the eye on the page for my shark was crossed out. How can that be? I did not know this page until today.

**Reply by IvanPotgieter01** [reply]
I had the same problem as you. No idea what caused it, but at least the issue seems fixable with the link.
